src/map/latent_effect_container.cpp

```diff
@@ -788,9 +788,9 @@ bool CLatentEffectContainer::ProcessLatentEffect(CLatentEffect& latentEffect)
         {
             size_t partyCount = 0;
             size_t trustCount = 0;
-            if (m_POwner->PParty != nullptr)
+            auto PLeader = (CCharEntity*)m_POwner->PParty->GetLeader();
+            if (PLeader != nullptr && m_POwner->PParty != nullptr)
```

Comment:
`m_POwner->PParty` must be checked for null before you can call a function on it

I don't particularly like this but

```
            if (m_POwner->PParty && dynamic_cast<CCharEntity*>(m_POwner->PParty->GetLeader()))
            {
                auto PLeader = static_cast<CCharEntity*>(m_POwner->PParty->GetLeader());
                trustCount = PLeader->PTrusts.size();
                partyCount = m_POwner->PParty->members.size();
            }
```